Ana Berdiales & Reinier Lasso/problema1.c

```diff
@@ -0,0 +1,12 @@
+// Programa que imprime los numeros de 2 en 2, de 3 en 3, de 4 en 4... hasta de 25 en 25.
+#include <stdio.h>
+void main ()

```

Comment:
Este programa no hace lo que se solicito

Reply:
debe imprimir utiilizando el while o do while o el for, los numero primero de 2 en 2, osea 2,4,6,8,10, luego de 3 en 3, osea 3,6,9,12, hasta 25, donde seria solamente 25